kujira/assetlist.json

```diff
@@ -87,6 +87,27 @@
         "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/kujira/images/mnta.svg",
         "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/kujira/images/mnta.png"
       }
+    },
+    {
+      "description": "The content house of Kujira",
+      "denom_units": [
+        {
+          "denom": "factory/kujira12cjjeytrqcj25uv349thltcygnp9k0kukpct0e/uwink",
+          "exponent": 0
+        },
+        {
+          "denom": "wink",
+          "exponent": 6
+        }
+      ],
+      "base": "factory/kujira12cjjeytrqcj25uv349thltcygnp9k0kukpct0e/uwink",
+      "name": "WINK",
+      "display": "WINK",
+      "symbol": "WINK",
+      "logo_URIs": {
+        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/kujira/images/wink.svg",
```

Comment:
can we reduce (best eliminate) the padding on the svg file? as it would make the token appear very small on displays